Upstream-Status: Pending

build fix for rpm5

8/19/2010 - created by Qing He <qing.he@intel.com>

diff --git a/zypp/target/rpm/BinHeader.cc b/zypp/target/rpm/BinHeader.cc
index e9ad382..86a9c96 100644
--- a/zypp/target/rpm/BinHeader.cc
+++ b/zypp/target/rpm/BinHeader.cc
@@ -19,7 +19,7 @@ extern "C"
 
 #ifndef _RPM_4_4_COMPAT
 #ifdef _RPM_5
-typedef rpmuint32_t rpm_count_t;
+typedef uint32_t rpm_count_t;
 #else
 typedef int32_t rpm_count_t;
 #endif
diff --git a/zypp/target/rpm/librpmDb.cv3.cc b/zypp/target/rpm/librpmDb.cv3.cc
index bf18e45..5454ea9 100644
--- a/zypp/target/rpm/librpmDb.cv3.cc
+++ b/zypp/target/rpm/librpmDb.cv3.cc
@@ -14,7 +14,7 @@ extern "C"
 {
 #ifndef _RPM_4_4_COMPAT
 #ifdef _RPM_5
-typedef rpmuint32_t rpm_count_t;
+typedef uint32_t rpm_count_t;
 #else
 typedef int32_t rpm_count_t;
 #endif
